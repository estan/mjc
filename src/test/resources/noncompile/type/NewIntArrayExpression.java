class NewIntArrayExpressionTest {
    public static void main(String[] args) {
        int i;
        int[] ia;
        boolean b;
        A cA;

        //ia = new int[i]; // OK!
        ia = new int[ia]; // INVALID_SIZE_TYPE
        ia = new int[b]; // INVALID_SIZE_TYPE
        ia = new int[cA]; // INVALID_SIZE_TYPE
    }
}
class A {}
