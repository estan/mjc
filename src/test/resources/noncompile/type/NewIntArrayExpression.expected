INVALID_SIZE_TYPE
INVALID_SIZE_TYPE
INVALID_SIZE_TYPE
